src/vos/vos_obj_cache.c

```diff
@@ -185,10 +198,18 @@ vos_obj_cache_current(void)
 	return vos_obj_cache_get();
 }
 
+static __thread struct vos_object	 obj_local = {0};
+
```

Comment:
into tls?

Reply:
is there an advantage to tls vs __thread ?

Reply:
oh, not actually, but we suppose to put per thread stuff to tls? anyway, it is not a big deal for the moment.